exercises/edward_brown/ex01_basics/src/WordCharPredicates.cpp

```diff
@@ -0,0 +1,5 @@
+#include <cctype>
+
+bool isWordChar(char Candidate) { return std::isspace(Candidate) == 0; }
+
+bool isWhitespaceChar(char Candidate) { return std::isspace(Candidate) != 0; }
```

Comment:
Do you test for the hyphen? See [here](http://www.mantidproject.org/New_Starter_C++_introduction#Exercise)